### PR TITLE
Timedesc and related libs 0.8.0

### PR DESCRIPTION
**Pull request by darrenldl**
OCaml date time handling library

Homepage: https://github.com/daypack-dev/timere

### CODE REVIEW

**Comment by darrenldl** [comment]
I will reopen the PR again when I've fixed the the errors reported by CI.

File: packages/timedesc-tzlocal-js/timedesc-tzlocal-js.0.8.0/opam [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+description: ""
+maintainer: ["Darren Ldl <darrenldldev@gmail.com>"]
+authors: ["Daypack developers"]
+license: "MIT"
+homepage: "https://github.com/daypack-dev/timere"
+bug-reports: "https://github.com/daypack-dev/timere/issues"
+depends: [
+  "ocaml" {>= "4.08.1"}
+  "dune" {>= "2.9"}
+  "timedesc-tzlocal" {= "0.8.0"}
```
Comment (kit-ty-kate) [inline packages/timedesc-tzlocal-js/timedesc-tzlocal-js.0.8.0/opam:13]: For next time, know that you can replace this by this to make it more clear it comes from the same project and make maintenance easier
```suggestion
  "timedesc-tzlocal" {= version}
```

Comment (darrenldl) [reply packages/timedesc-tzlocal-js/timedesc-tzlocal-js.0.8.0/opam:13]: Oh that's very neat! I'll push a fix for this. Thanks!

Comment (darrenldl) [reply packages/timedesc-tzlocal-js/timedesc-tzlocal-js.0.8.0/opam:13]: Okay found section on interpolation within string via `%{...}%`

**Comment by darrenldl** [comment]
Is the fedora(?) build failure something to do with CI or is it something I should investigate?

**Comment by darrenldl** [comment]
Ah I see, makes sense. The timing is not a problem in any case.

Thanks for the reply!

**Pull request merged**